.dotnet/Azure.AI.OpenAI.Staging/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenAI;
+
+namespace Azure.AI.OpenAI.Staging;
+
+public partial class AzureOpenAIClientOptions : OpenAIClientOptions
+{
+    public string ApiVersion { get; }
```

Comment:
This should use the standard Azure SDK way of doing this. See some existing Azure SDK track 2 library, i.e. the ServiceVersion enum.

Reply:
Merged from the existing azure-sdk-for-net library.